index.html

```diff
@@ -14,183 +14,7 @@
 
     <title>Frontend Mentor | Fylo landing page with two column layout</title>
     <link rel="stylesheet" href="style.css" />
-    <style>
-      .attribution {
-        font-size: 11px;
-        text-align: center;
-      }
-      .attribution a {
-        color: hsl(228, 45%, 44%);
-      }
-    </style>
   </head>
   <body>
-    <header>
-      <div class="icon">
-        <img src="./images/logo.svg" alt="Logo" />
-      </div>
-      <nav class="navbar">
-        <div class="hamburger-icon">
-          <img src="./images/menu-hamburger.svg" alt="" onclick="openMenu()" />
-        </div>
-        <div class="menu" id="menu">
-          <ul>
-            <li><a href="#">Features</a></li>
-            <li><a href="#">Team</a></li>
-            <li><a href="#">Sign In</a></li>
-          </ul>
-        </div>
-      </nav>
-    </header>
```

Comment:
Great HTML structure @askinekincihf 
Your code is very well organised, you use semantic HTML when you can and everything is nicely structured 👍